optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -512,9 +513,12 @@ private BarRenderer createBarChartRenderer(NumberFormat numberFormat) {
     private XYPlot createScalabilityPlot(List<XYSeries> seriesList,
             String xAxisLabel, NumberFormat xAxisNumberFormat,
             String yAxisLabel, NumberFormat yAxisNumberFormat) {
-        NumberAxis xAxis = new NumberAxis(xAxisLabel);
+        NumberAxis xAxis;
         if (useLogarithmicProblemScale(seriesList)) {
-            xAxis = new LogarithmicAxis(xAxis.getLabel() + " (logarithmic)");
+            xAxis = new LogarithmicAxis(xAxisLabel + " (logarithmic)");
+            ((LogarithmicAxis)xAxis).setAllowNegativesFlag(true);
```

Comment:
I agree with your comment in your mail, having a separate variable is indeed better than casting. I didn't realize it was a subclass specific method.
Call the separate variable name logarithmicAxis, that way it's clear it's a LogarithmicAxis specific method. There's no need to prefix that variable with x, because it's scoped within that if method, so it's very local.

Anyway, nitpicking here :)